### ISSUE TITLE
Implement email when user earns a STEAM badge

### ISSUE THREAD

**Issue by emgollie** [mozilla/openbadger#197]
Per https://github.com/mozilla/openbadger/issues/166#issuecomment-19041822. cc @threeqube 

Hi [earnername],

Congrats! You've earned the [Badge name] Badge for great work in [STEAM field].

You were awarded this badge because you earned it. It's the result of your hard work.

Now you can try more fun city-level challenges at iRemix here: [iRemix LINK]

Keep on earning and exploring!
From your friends at the Chicago Summer of Learning

Linked fields:
[earnername]
[Badge name]
[STEAM field]
[iRemix LINK]

**Comment by iamjessklein** [comment]
:+1:

**Comment by brianloveswords** [comment]
@toolness, I think this is going in CSOL isn't it?

**Comment by cmcavoy** [comment]
yeah, not sure if this is in CSOL or in openbadger given the new emailing framework that @toolenss and @christensenep built. Assigning to @christensenep

**Comment by toolness** [comment]
Yeah, it'll go into CSOL, though #191 is required to make that happen.

**Comment by ghost** [comment]
So in the case where a learner is awarded a STEAM badge, but already has earned one previously and signed up at iremix, we want to send an email to them that doesn't include an iremix invitation link.  I have a placeholder email template that fills this role, but it's just the same copy as the email that includes the link with that line removed.  Please let me know if you want something other than that and I will change it.

**Comment by emgollie** [comment]
@christensenep I think that's fine. In that case, I would just hyperlink "Chicago Summer of Learning" in the end signoff so they can easily come back to the site. Thanks!
